rollup.config.js

```diff
@@ -1,50 +1,19 @@
-import resolve from '@rollup/plugin-node-resolve';
-import babel from '@rollup/plugin-babel';
-import commonjs from '@rollup/plugin-commonjs';
-import json from '@rollup/plugin-json';
-import { visualizer } from 'rollup-plugin-visualizer';
+import replace from '@rollup/plugin-replace';
```

Comment:
maybe rename this file to rollup.dev.config.js, so it will not be miss used in the future